applications/chainstaff.md

```diff
@@ -0,0 +1,195 @@
+- [chainstaff.info](https://www.chainstaff.info/)
+
+### Team's experience
+
+Chainstaffs CEO & Founder Alessandro is an experienced HR professional with a CAS in Human-Centered Organization from Berner Fachhochschule (BFH). He spent over eight years at Swisscom, Switzerland’s leading telecom company, where he spent over 8 years as HR Operations Specialist Then he joined MCH Group as an HR Project Specialist, and later, he tackled a three-year stint as an HR expert at SBB CFF FFS, Switzerland's national railway company. He is the CEO and founder of Chainstaff.
+
+SayNode stands out for having the tech stack that combines blockchain and mobile development. We strongly believe that perfect user experience will be the key for a successful Web3 project. The team around SayNode has already implemented various projects in the Web3 area. Regarding Vechain specifically, a grant for the Dev-Thorkit-Dart has already been granted and this project will include the same developers. The team has also seen their first milestone delivery approved for a [V3 style mobile swap on Vechain](https://github.com/vechain/grant-program/blob/master/applications/mobile-v3-swap.md).
+
+In addition, SayNode already deployed two Flutter Apps, which have been accepted in both Apple Store and Android Store. Overall, the team has experience with DAO creation, wallet creation, tokens and multichain tokens, ICOs and everything around App-Development.
+
+### Team Code Repos
+
+In addition to the project and team repos mentioned below, we would also like to focus on https://raclette.finance/ which was created by our team. For confidentiality agreements no repo can be released for this project yet. As soon as the audit is complete, this will be made public.
+
+- https://github.com/SayNode/thor-devkit.dart
+- https://github.com/SayNode/thor-request.dart
+- https://github.com/SayNode/DAO-VeChain
+- https://github.com/SayNode/axper
+- https://github.com/SayNode/supply_chain_resilience_final
+- https://github.com/SayNode/Vidaia
+- https://github.com/RsTs23
+- https://github.com/wernerliechti
+- https://github.com/YannMarti
+- https://github.com/paulamstutz
+- https://github.com/Krfld
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/alessandro-pizzoferrato-1680a9107/
+- https://www.linkedin.com/in/renato-sch%C3%A4r-8a9a19a5/
+- https://www.linkedin.com/in/werner-liechti-568b19213/
+- https://www.linkedin.com/in/paula-amstutz/
+- https://www.linkedin.com/in/joaogmorais/
+- https://www.linkedin.com/in/simon-heer-202866110/
+- https://www.linkedin.com/in/krfld/
+
+## Development Roadmap
+
+### Overview
+
+|                            | Milestone 1 | Milestone 2 | Total  |
+| -------------------------- | ----------- | ----------- | ------ |
+| Estimated Duration         | 44 d        | 20 d        | 64 d   |
+| Full-time equivalent (FTE) | 2           | 1           | 3      |
+| Cost (up to $ 30,000)      | 20,000      | 10,000      | 30,000 |

```

Comment:
Please be aware that the funds for milestone 1 should not exceed 40% of the total amount you are requesting. It is advisable to divide your milestone 1 into two separate parts.